### ISSUE TITLE
Ossia.client instacrash

### ISSUE THREAD

**Issue by petervanhaaften** [ossia/libossia#607]
Hello, I have a new issue that I imagine is connected in the same way to issue #606 : [https://github.com/ossia/libossia/issues/606](url)

**Observed Behavior**

If I make a patch with 'ossia.namespace.model' together with the 'ossia.client' object, save it, and reload the patch: Max instacrashes.

**Steps to reproduce**

1. Create new patch in Max/MSP
2. Add 'ossia.namespacebrowser.model' [ from: https://github.com/evanmtp/tml-ossia/tree/master/utilities/namespacebrowser ]
3. Add 'ossia.client' object
4. Save patch & close max.
5. Re-open patch [ crash ]

**Notes**

Crash log attached.

Ossia version: "4c31e19c8"
MacOS: High sierra 10.13.6
[crashlog1.txt](https://github.com/ossia/libossia/files/5533633/crashlog1.txt)

**Comment by petervanhaaften** [comment]
Navid is having the same issue on his computer macOS 10.14.6. Crash log attatched. Thank you for any help:)
[navnamespacecrashlog.txt](https://github.com/ossia/libossia/files/5533651/navnamespacecrashlog.txt)

**Comment by petervanhaaften** [comment]
Okay! Positive info to report!

I created a new patch with ossia.namespacebrowser.model and ossia.client only. Saved, instacrash. Crash log attached

Then: I removed [ossia.fuzzy] from ossia.namespacebrowser.model, saved it. Now, no instacrash!

We need to do more testing over next days. But seems likely it is a problem with [ossia.fuzzy]. So good to note and explore further. Thank you!

**Comment by evanmtp** [comment]
(ossia.fuzzysearch, that is)

**Comment by avilleret** [comment]
@petervanhaaften could you share the patcher that crashes on your side ? with its deps ? 
I can't reproduce the crash here with Max 8.1.8 and macOS 11.0.1 and ossia-max 4c31e19c

**Comment by avilleret** [comment]
@petervanhaaften & @evanmtp could you test with https://github.com/ossia/libossia/releases/download/v1.1.0-a1/ossia-max-osx.zip ?
@jcelerier pointed out some line in `ossia.search` (formely known as `ossia.fuzzysearch`) that might lead to crash.

**Comment by evanmtp** [comment]
Thanks @avilleret ! We'll check it out over the course of the next week. Today is a studio day for @petervanhaaften and @navid , and I'm working on another project until Sunday, so I'd say we're most likely to look at this next Wednesday or Thursday. Just checking, I see that https://github.com/ossia/libossia/releases/tag/v1.1.0-a2 is out now - do you want us to check with that version, or still v1.1.0-a1 ?

**Comment by jcelerier** [comment]
@evanmtp I think that a good rule of thumb is to always use the latest :p

**Comment by evanmtp** [comment]
Agreed! Juuuust checking :D

**Comment by petervanhaaften** [comment]
Hi thank you we are still trying to pinpoint the issue, but we are definitely having problems with some element inside namespacebrowser---which I understand now that Evan wrote so I should discuss my issues with him first. Basically, putting namespacebrowser in a patch with an ossia module that utilizes circular communication causes outputs from other ossia modules to break. Anyway.. . . to be investigated further.

But I tried https://github.com/ossia/libossia/releases/download/v1.1.0-a1/ossia-max-osx.zip and it causes max to instacrash. I am on High Sierra---is this built for High Sierra? As usual in sound land we stay a few versions behind because the rapid MacOs changes cause many important tools to break.

I did not see a build for max here: https://github.com/ossia/libossia/releases/tag/v1.1.0-a2 

Thank you, and until the next time I have a moment to pop in here:)
